signer/src/transaction_coordinator.rs

```diff
@@ -169,6 +177,12 @@ where
             .await?
             .ok_or(Error::NoChainTip)?;
 
+        if self.needs_dkg(&bitcoin_chain_tip).await? == DkgState::NeedsDkg {
+            // This function returns the new DKG aggregate key. That
+            // aggregate key is different from aggregate key of the signers.
```

Comment:
qq: just to be clear on the notation, the `aggregate key of the signers` is the stacks multisig principal?

Reply:
Well not exactly. The stacks multi-sig principal is formed differently. It does not use the aggregate key of the public keys embedded in the address. I added this line here because I wanted to be clear that this will be different from the aggregate key of the signers, even though we associate it with the signers' public keys.